bindings/nodejs/lib/types/block/core/wrapper.ts

```diff
@@ -73,6 +75,16 @@ class BlockWrapper {
         this.block = block;
     }
 
+    /**
+     * Compute the block ID (Blake2b256 hash of the block bytes).
+     *
+     * @param params The network protocol parameters.
+     * @returns The corresponding block ID.
+     */
+    id(params: ProtocolParameters): BlockId {
```

Comment:
Is this another scenario where we're going to be importing stuff we don't need if you just want the types?

Reply:
Seems to be so yes. :/